File: python/claude/p116.py
import os

# Using format string to construct the ls command
directory = "/home/user"

# Using string format method to create the command string
# (similar to print function with placeholders)
command = "ls -l {}".format(directory)

# Execute the command using os.system function
os.system(command)

# Alternative using f-string syntax
print("Using f-string:")
command_f = f"ls -l {directory}"
os.system(command_f)

File: c/claude/p116.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>

int main() {
    // Using format string to construct the ls command
    char directory[256] = "/home/user";
    char command[512];
    
    // Using sprintf (similar to print with format string)
    // to create the command string
    sprintf(command, "ls -l %s", directory);
    
    // Execute the command using system function
    system(command);
    
    return 0;
}
